packages/contracts-bedrock/test/L1/OPStackManager.t.sol

```diff
@@ -32,42 +32,58 @@ contract OPStackManager_Harness is OPStackManager {
 // we can use its setup to deploy the implementations similarly to how a real deployment would
 // happen.
 contract OPStackManager_Deploy_Test is DeployOPChain_TestBase {
+    using stdStorage for StdStorage;
+
+    function setUp() public override {
+        DeployOPChain_TestBase.setUp();
+
+        doi.set(doi.opChainProxyAdminOwner.selector, opChainProxyAdminOwner);
+        doi.set(doi.systemConfigOwner.selector, systemConfigOwner);
+        doi.set(doi.batcher.selector, batcher);
+        doi.set(doi.unsafeBlockSigner.selector, unsafeBlockSigner);
+        doi.set(doi.proposer.selector, proposer);
+        doi.set(doi.challenger.selector, challenger);
+        doi.set(doi.basefeeScalar.selector, basefeeScalar);
+        doi.set(doi.blobBaseFeeScalar.selector, blobBaseFeeScalar);
+        doi.set(doi.l2ChainId.selector, l2ChainId);
+        doi.set(doi.opsm.selector, address(opsm));
+    }
+
     // This helper function is used to convert the input struct type defined in DeployOPChain.s.sol
     // to the input struct type defined in OPStackManager.sol.
```

Comment:
Do we want to keep the OPSM struct?

Reply:
It's very likely that we'll have to, because I think we'll get stack too deep with that many inputs to a function. I'll play around with this a bit and open a PR if any good patterns emerge